src/module-info.json

```diff
@@ -0,0 +1,53 @@
+{
+  "coreModules": [],
+  "dependenceModules": [
+    {
+      "name": "linagora.esn.calendar",
```

Comment:
This is not correct for this SPA

Reply:
You are right, this module does not depend on any old modules. So I removed this file